apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -246,9 +246,9 @@ const FundForm = ({
     const today = new Date()
     const zeroErrArray = []
     const dateErrArray = []
-    Object.values(bounties).forEach(bounty => {
-      if (bounty.payout === 0) zeroErrArray.push(bounty.issueId)
-      if (today > bounty.deadline) dateErrArray.push(bounty.issueId)
+    issues.forEach(issue => {
+      if (Number(bounties[issue.id].payout) === 0) zeroErrArray.push(issue.id)
+      if (today > bounties[issue.id].deadline) dateErrArray.push(issue.id)
```

Comment:
This looks like it may have changed because of a merge conflict. I think we want the `bounties` version.